client/i18n/locales/english/intro.json

```
@@ -592,7 +592,7 @@
         "title": "MongoDB and Mongoose",
         "intro": [
           "MongoDB is a database application that stores JSON documents (or records) that you can use in your application. Unlike SQL, another type of database, Mongo is a non-relational or \"NoSQL\" database. This means Mongo stores all associated data within one record, instead of storing it across many preset tables as in a SQL database.",
```

Another suggestion if we decide to change "Mongo" to "MongoDB":

```suggestion
          "MongoDB is a database application that stores JSON documents (or records) that you can use in your application. Unlike SQL, another type of database, MongoDB is a non-relational or \"NoSQL\" database. This means MongoDB stores all associated data within one record, instead of storing it across many preset tables as in a SQL database.",
```